components/layout/WidgetGrid/WidgetGrid.stories.jsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import WidgetGrid from ".";
+
+const meta = {
+  component: WidgetGrid,
+};
+export default meta;
+
+const MockWidget = styled.div`
```

Comment:
this example shows how to set up responsive grid columns, if a widget is configured with a span greater than the current number of columns, then CSS `min()` will compare the number of columns from the prop and the number of columns defined by the grid and use the lesser value.